kmcd.md

```diff
@@ -72,6 +72,115 @@ State Storage/Revert Semantics
+    rule contract(Deploy . _) => Deploy
+
+    syntax DeployStep ::= "deploy" Address
+ // --------------------------------------
+    rule <k> Deploy . deploy GOV
+          => call Deploy . deployVat
+          ~> call Deploy . deployDai
+          ~> call Deploy . deployTaxation
+          ~> call Deploy . deployAuctions GOV
+          ~> call Deploy . deployLiquidator
+          ~> call Deploy . deployShutdown
+         ...
+         </k>
+
+    syntax DeployStep ::= "deployVat"
+ // ---------------------------------
+    rule <k> Deploy . deployVat
+          => call Vat  . constructor
+          ~> call Spot . constructor Vat
+          ~> call Vat  . rely Spot
+         ...
+         </k>
+
+    syntax DeployStep ::= "deployDai"
+ // ---------------------------------
+    rule <k> Deploy . deployDai
+          => call Dai     . constructor
+          ~> call DaiJoin . constructor Vat Dai
+          ~> call Dai     . rely DaiJoin
+         ...
+         </k>
+
+    syntax DeployStep ::= "deployTaxation"
+ // --------------------------------------
+    rule <k> Deploy . deployTaxation
+          => call Jug . constructor Vat
+          ~> call Pot . constructor Vat
+          ~> call Vat . rely Jug
+          ~> call Vat . rely Pot
+         ...
+         </k>
+
+    syntax DeployStep ::= "deployAuctions" Address
+ // ----------------------------------------------
+    rule <k> Deploy . deployAuctions GOV
+          => call Flap . constructor Vat GOV
+          ~> call Flop . constructor Vat GOV
+          ~> call Vow  . constructor Vat Flap Flop
+          ~> call Jug  . file vow-file Vow
+          ~> call Pot  . file vow-file Vow
+          ~> call Vat  . rely Flop
+          ~> call Flap . rely Vow
+          ~> call Flop . rely Vow
+         ...
+         </k>
+
+    syntax DeployStep ::= "deployLiquidator"
+ // ----------------------------------------
+    rule <k> Deploy . deployLiquidator
+          => call Cat . constructor Vat
+          ~> call Cat . file vow-file Vow
+          ~> call Vat . rely Cat
+          ~> call Vow . rely Cat
+         ...
+         </k>
+
+    syntax DeployStep ::= "deployShutdown"
+ // --------------------------------------
+    rule <k> Deploy . deployShutdown
+          => End  . constructor
+          ~> End  . file vat-file  Vat
+          ~> End  . file cat-file  Cat
+          ~> End  . file vow-file  Vow
+          ~> End  . file pot-file  Pot
+          ~> End  . file spot-file Spot
+          ~> Vat  . rely End
+          ~> Cat  . rely End
+          ~> Vow  . rely End
+          ~> Pot  . rely End
+          ~> Spot . rely End
+         ...
+         </k>
+
+    syntax DeployStep ::= "deployCollateral" String Address
+ // -------------------------------------------------------
+    rule <k> Deploy . deployCollateral ILK_ID JOIN
+          => call Flip ILK_ID . constructor Vat
+          // ~> call Spot . file pip ILK_ID PIP
```

Comment:
Is this commented out because we don't model the `pips` as contracts?

Reply:
Yes, that's correct.